torch/distributed/tensor/parallel/api.py

```diff
@@ -80,6 +80,8 @@ def parallelize_module(  # type: ignore[return]
         granularity, you need to pass in a dict of module FQN and parallel style instead.
     """
 
+    torch._C._log_api_usage_once("torch.distributed.tensor.parallel.parallelize_module")
```

Comment:
same question: I think it should only be called inside the API instead of in a global scope?

Reply:
Looks like they are 4 spaces same as L85. Or unless I am missing anything here?

Reply:
hmmm nvm, looks like github UI shows it's a different indention when my screen is small.